version/src/lib.rs

```diff
@@ -26,6 +26,8 @@ pub struct Version {
     pub minor: u16,
     pub patch: u16,
     pub commit: Option<u32>, // first 4 bytes of the sha1 commit hash
+    pub devbuild: bool,
```

Comment:
Changing this structure actually affects the bits that are sent over gossip

Reply:
Thanks for the heads up! I'll dig a little deeper to understand how this is used.